trackintel/preprocessing/staypoints.py

```diff
@@ -169,3 +194,20 @@ def generate_locations(
     ret_loc["user_id"] = ret_loc["user_id"].astype(ret_stps["user_id"].dtype)
 
     return ret_stps, ret_loc
+
+
+def _generate_locations_per_user(user_staypoints, distance_metric, db, geo_col):
+    """function called after groupby: should only contain records of one user; see generate_locations() function for parameter meaning."""
```

Comment:
This line is too long. We limit the line limit to 120, [here ](https://github.com/mie-lab/trackintel/blob/master/CONTRIBUTING.md#others) Black will not format comments

Reply:
done